### ISSUE TITLE
readImageFile method is very slow compare to python API.

### ISSUE THREAD

**Issue by zhuczhuc** [InsightSoftwareConsortium/itk-js#294]
I tried to load nrrd data, and I succeeded. But later, I feel the speed is very slow compare to python code.  It takes about 5 seconds loading a 60M gzip image data. But when I used simpleITK readImage() method, I never feel waiting a lot. Here is my code: 
```
function handleFile(event) {
  const dataTransfer = event.dataTransfer;
  const files = event.target.files || dataTransfer.files;

  console.time('read image');
  readImageFile(null, files[0]).then(({ image: itkImage, webWorker }) => {
    console.timeEnd('read image');
    webWorker.terminate();
    const vtkImg = vtkITKHelper.convertItkToVtkImage(itkImage);
  });
}
```
Is that normal or my browser's reason?

**Comment by thewtex** [comment]
5 seconds seems long.

A few questions:

- What is the browser version?
- What is the performance in Firefox and Chrome?
- How does the speed of a non-gzipped NRRD image compare?

**Comment by zhuczhuc** [comment]
I'm using Chrome with version 80.0.3987.132 64bit. 
The time cost about 600 ms when reading the non-gzipped NRRD image, same image file and the size is 90 M compare to 60 gzipped image. 
Also, I tried to do the same thing with nii format image. I can see the speed become 10 times longer.
I think I have found the problem. 

Very thanks. And how could it become faster when I use itkjs to read gzipped image.
